### ISSUE TITLE
Remove duplicate ID on Disability Page from CMS team

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-team#56778]
## What does your team need support for? Check all that apply.
- [ ] Launching one or more unauthenticated pages, online forms, apps, or tools on VA.gov
- [x] Making revisions to one or more existing VA.gov pages
- [ ] Something else 
 
An accessibility error regarding a duplicate ID was flagged today (4/12) on a Disability benefits page: [Evidence needed](https://www.va.gov/disability/how-to-file-claim/evidence-needed/).

There are 2 IDs that were added into rich text editors that are duplicates:

1. Under Type of claim heading > Supplemental claim accordion
   1. The text that says "You’ll need to submit or identify these types of evidence:"
![image](https://user-images.githubusercontent.com/106678594/231543801-4824c9f9-1ce9-4bba-82e8-7bd2d9e0e5fa.png)
2. Under Additional evidence heading > Presumptive condition accordion
   1. The bold text that says "You’ll need to submit or identify these types of evidence:"
![image](https://user-images.githubusercontent.com/106678594/231543489-bc1f6fff-24fd-4397-b13d-1eb07f63bb0d.png)

Both of these IDs should be removed from the editor in n order to resolve the accessibility issue. This issue will continue to be flagged in the daily scan until it is resolved.

Also, headings and IDs were discussed earlier in a [slack thread here](https://dsva.slack.com/archives/C01K37HRUAH/p1670249085485899) IDs are automatically added to headings but when manually added, this causes a high chance of duplicates. These duplicates are also often accidently caused by copy/pasting content into the text editor so that is something to be careful of as well.

## Will this new product be released incrementally (for instance 25% of users initially)? 
  - [ ] Yes
  - [ ] No
 
**Note:** If you checked yes, we'll reach out to discuss details about the content in the react widget.
  
## When do you expect to launch your product to 100% of users?
  
*Please provide an estimated date so our team can create other relevant tickets.*
  
## Supporting artifacts
*Please provide supporting artifacts as available.*

- Link to product outline:
- Link to or attach user flows:
- Link to prototype or mockups:
- Link to any epics/issues that may be helpful:

## Will this work be going through the Collaboration Cycle?
- [ ] Yes 
- [ ] No

## When does this work need to be done?
- Estimated launch date:
- Estimated staging review date:
- Content, Accessibility, and IA work needed by: 

## Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [ ] No, let's work asynchronously and meet if needed

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team

- Team name: 
-	OCTO-DE product owner: Dave Conlon
-	Product manager: 
-	Designer: 
-	FE engineer: 
-	Product/team Slack channel:

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag Randi Hecht.**

If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)

**Comment by megzehn** [comment]
Thanks, @RLHecht and @laflannery! I removed the duplicate IDs in both places.

@katherine-fung, FYI, I reverted back to the draft with your MST changes and removed the duplicate IDs on that version as well.

**Comment by laflannery** [comment]
@megzehn code looks good! I will confirm tomorrow afternoon when the scan runs but I don't expect an issue

**Comment by laflannery** [comment]
@megzehn and @RLHecht scan looks good - this ticket can be closed, thanks!

**Comment by RLHecht** [comment]
Thanks! Closing!
